js/miss.js

```diff
@@ -0,0 +1,32 @@
+
+
+
+function onMissLaser() {
+    makeNoise('miss')
+    changeScore(-5)
+
```

Comment:
whitespace

Reply:
Removed all unnecessary whitespace in ad42ddf.